### ISSUE TITLE
False-positive E1121:Too many positional arguments for random.choice

### ISSUE THREAD

**Issue by TommyB1992** [PyCQA/pylint#2779]
### Steps to reproduce
1.  Create a file with:
```
"""aaa"""
import random

TEST = ["a"]
A = random.choice(TEST, 1)
```

2. execute on terminal: pylint filename 
3.

### Current behavior
E: 42,16: Too many positional arguments for method call (too-many-function-args)

### pylint --version output
pylint 1.8.4, 
astroid 1.6.3
Python 3.6.5 |Anaconda custom (64-bit)| (default, Apr 29 2018, 16:14:56) 
[GCC 7.2.0]

Thank you

**Comment by lburg** [comment]
Hi, I reproduce the same issue with another code snippet, on a more recent pylint version : 

```python
import functools                     
                                     
def func(a, b):                      
    print(a, b)                      
                                     
func_b = functools.partial(func, 'a')
func_b('b')                          
func('a', 'b')                       
```

pylint complains on line 8, `func('a', 'b')`. I expect no errors from this code.

pylint --version output
===

pylint 2.2.2
astroid 2.1.0
Python 3.6.6 (default, Sep 13 2018, 13:48:10)
[GCC 8.2.1 20180831]

**Comment by TommyB1992** [comment]
Yes that was a my error.

Sorry to waste your time.

**Comment by PCManticore** [comment]
The error seems okay to me, there's actually too many arguments passed there.

@lburg Please open a separate issue as it's not related to this one. Even if it is the same error, the root cause can be different as each stdlib module can have or not various support in our inference engine.

**Comment by lburg** [comment]
@PCManticore I was about to, but using pylint and astroid preview version seems to have fixed it. Thanks!
